test/BUILD

```diff
@@ -393,6 +393,15 @@ scala_junit_test(
     suffixes = ["Test"],
 )
 
+scala_specs2_junit_test(
+    name = "JunitSpecs2Test",
```

Comment:
why do we need this target? It's identical to Specs2Tests target in this file

Reply:
You're right, must have missed it!